redbot/cogs/trivia/trivia.py

```diff
@@ -402,6 +405,64 @@ async def trivia_list(self, ctx: commands.Context):
             else:
                 await ctx.send(msg)
 
+    @trivia.command(name="info")
+    async def trivia_info(self, ctx: commands.Context, category: str.lower):
+        """Get information about a trivia category."""
+        try:
+            data = self.get_trivia_list(category)
+        except FileNotFoundError:
+            return await ctx.send(
+                _(
+                    "Invalid category `{name}`. See `{prefix}trivia list` for a list of "
+                    "trivia categories."
+                ).format(name=category, prefix=ctx.clean_prefix)
```

Comment:
- Adding formatting to dynamic strings should preferably be done with the use of the formatting helpers
- Commands in the translatable string should preferably be put using a format field so that they don't get translated
- I think that "Invalid category {name}." could be reworded to "Category {name} does not exist." in order to make it clearer
```suggestion
                _(
                    "Category {name} does not exist."
                    " See {command} for the list of available trivia categories."
                ).format(name=inline(category), command=inline(f"{ctx.clean_prefix}trivia list"))
```